docs/sources/next/using-k6/javascript-compatibility-mode.md

```diff
@@ -1,44 +1,71 @@
 ---
-title: JavaScript Compatibility Mode
+title: JavaScript compatibility mode
+menuTitle: JavaScript mode
 excerpt: 'k6 supports running test scripts with different ECMAScript compatibility modes using --compatibility-mode'
 _build:
   list: false
 weight: 19
 ---
 
-# JavaScript Compatibility Mode
+# JavaScript compatibility mode
 
-You can run test scripts with different ECMAScript compatibility modes with the
-`run --compatibility-mode` CLI option or `K6_COMPATIBILITY_MODE` environment variable.
+You can write k6 scripts in various ECMAScript versions:
 
-Currently two modes are available:
+- ES6+ JavaScript with ES modules (ESM).
+- Plain old JavaScript (ES5.1+) with CommonJS modules.
 
-## Base
+k6 supports both module types and most ES6+ features in all k6 execution modes: local, distributed, and cloud.
+
```

Comment:
I would really prefer for us to have a warning/info/advice to *not* care/do/or further read about this in case they do not have problems with start up times. 

It really shouldn't be needed in like 99% of the cases and hopefully soon in none of the cases and will actually be a thing I would higly recommend people not to do. But this is after it is supported natively

Reply:
Fixed by https://github.com/grafana/k6-docs/pull/1489/commits/09e692537ac3c25488ca3049a95309ef1b0fa574

- removing `When executing large scripts, the k6 Babel transformation can slightly delay the startup time.`
- Already included: `Generally, this mode is not recommended as it offers minor benefits in reducing startup time.`